patternfly-a11y.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  aggregate: true,
+  prefix: 'http://localhost:9000',
+  ignoreRules: 'color-contrast,page-has-heading-one,scrollable-region-focusable',
```

Comment:
Why are we ignoring these rules?

Reply:
These rule were ignored before per @jessiehuff . `page-has-heading-one` was added because our fullscreen examples are not expected to have a page heading.

Reply:
I believe when the script was first written, we noticed strange false positives with color contrast. I've seen a few [open issues](https://github.com/dequelabs/axe-core/issues?q=is%3Aissue+is%3Aopen+color+contrast) with axe-core on color contrast so I'm not sure if it's an issue on our side or theirs, but it's worth revisiting. My preference is to have none of these ignored if we can find solutions to whatever problems we've run into in the past. In the PR description, you mentioned that "New tests fail due to a newer version of axe-core", do we have open issues for those? I'd love to get them fixed and turn this all back on as I would see that as high priority for accessibility.

Reply:
Once this PR is merged I'll open issues per the report: https://patternfly-pr-3178-coverage.surge.sh/